requirements.txt

```
@@ -6,7 +6,11 @@
 colorama
 cray-product-catalog
 PyYAML
+requests==2.25.1
 requests-retry-session
+kubernetes==26.1.0
+python-keycloak==0.27.1
```

In the requirements.txt file, we generally do not pin the version. The only times we should specify versions in this file is if they are true requirements (as in,. the code won't work without a certain minimum or maximum package version).

For cases where we just want to pin the version that we are choosing to use, that is what the constraints.txt file is for.

```suggestion
python-keycloak
```